mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientService.kt

```diff
@@ -39,6 +40,7 @@ public interface GitHubClientService {
      * Searches and returns user [Mention]s by [Username] that was mentioned
      * and their [PersonalAccessToken].
```

Comment:
Comma is missing before `and`.

Right now it reads as if we return PAT as well.

Let's rewrite:

```
Searches for user `Mentions` by the GitHub name of the user.

Uses `PersonalAccessToken` to access GitHub API.
```

And please document the parameters properly as well.